src/heroes/orisa/gallop.opy

```diff
@@ -9,84 +9,87 @@ globalvar orisaGallopDamage = createWorkshopSetting(int[0:200], "Orisa", "Time g
-    eventPlayer.HeroEffects.append(getLastCreatedEntity())
-    createEffect(getPlayers(getOppositeTeam(eventPlayer.getTeam())), Effect.GOOD_AURA, Color.RED, updateEveryTick(eventPlayer.getEyePosition()), 0.5, EffectReeval.VISIBILITY_POSITION_AND_RADIUS)
-    eventPlayer.HeroEffects.append(getLastCreatedEntity())
-    createEffect(getAllPlayers(), Effect.WINSTON_PRIMAL_RAGE, eventPlayer.getTeam(), updateEveryTick(eventPlayer.getEyePosition()), 1, EffectReeval.VISIBILITY_POSITION_AND_RADIUS)
-    eventPlayer.HeroEffects.append(getLastCreatedEntity())
-    createEffect(getAllPlayers(), Effect.ENERGY_SOUND, eventPlayer.getTeam(), updateEveryTick(eventPlayer.getEyePosition()), 500, EffectReeval.VISIBILITY_POSITION_AND_RADIUS)
-    eventPlayer.HeroEffects.append(getLastCreatedEntity())
-    playEffect(getAllPlayers(), DynamicEffect.DEBUFF_IMPACT_SOUND, eventPlayer.getTeam(), eventPlayer.getEyePosition(), 500)
-    playEffect(getAllPlayers(), DynamicEffect.ORISA_HALT_IMPLOSION_SOUND, eventPlayer.getTeam(), eventPlayer.getEyePosition(), 500)
-    #Casting
-    eventPlayer.HeroAbsoluteSpeedMod = 0
-    eventPlayer.startFacing(vect(0, 0, 1), 500, Relativity.TO_PLAYER, FacingReeval.DIRECTION_AND_TURN_RATE)
-    waitUntil(any([i == true for i in [eventPlayer.hasStatusEffect(Status.ASLEEP), eventPlayer.hasStatusEffect(Status.HACKED), eventPlayer.hasStatusEffect(Status.KNOCKED_DOWN), eventPlayer.hasStatusEffect(Status.STUNNED), eventPlayer.hasStatusEffect(Status.FROZEN)]]), orisaGallopDelay)
-    eventPlayer.setDamageDealt(100)
-    #Gallop
-    eventPlayer.startForcingThrottle(1, 1, 0, 0, 0, 0)
-    eventPlayer.HeroAbsoluteSpeedMod = 500
-    eventPlayer.disablePlayerCollision()
-    eventPlayer.forceButtonPress(Button.MELEE)
-    eventPlayer.Galloping = true
-    eventPlayer.ExcludedPlayers = []
-    eventPlayer.TimeRemaining = 0
-    chase(eventPlayer.TimeRemaining, 9999, rate=1, ChaseReeval.DESTINATION_AND_RATE)
-    while eventPlayer.TimeRemaining < orisaGallopDuration:
-        if any([i == true for i in [not isInLoS(eventPlayer.getEyePosition(), eventPlayer.getEyePosition() + worldVector(Vector.FORWARD * 2, eventPlayer, Transform.ROTATION), BarrierLos.PASS_THROUGH_BARRIERS), eventPlayer.hasStatusEffect(Status.ASLEEP), eventPlayer.hasStatusEffect(Status.HACKED), eventPlayer.hasStatusEffect(Status.KNOCKED_DOWN), eventPlayer.hasStatusEffect(Status.STUNNED), eventPlayer.hasStatusEffect(Status.FROZEN)]]):
-            #Stop Gallop
-            break
-        else:
-            eventPlayer.TargetPlayers = [player for player in getPlayersInRadius(eventPlayer.getEyePosition(), 5, getOppositeTeam(eventPlayer.getTeam()), LosCheck.SURFACES) if eventPlayer.isInViewAngle(player.getEyePosition(), 90)].exclude(eventPlayer.ExcludedPlayers)
-            damage(eventPlayer.TargetPlayers, eventPlayer, orisaGallopDamage)
-            eventPlayer.ExcludedPlayers.append(eventPlayer.TargetPlayers)
+    else:
```

Comment:
This is more of a code style nitpick than anything else, but since the preceding `if` statement already terminates execution (returns) if its condition is met, there is no need to have the `else`.